zwave_js_server/model/version.py

```diff
@@ -18,4 +20,6 @@ def from_message(cls, msg: dict) -> "VersionInfo":
             driver_version=msg["driverVersion"],
             server_version=msg["serverVersion"],
             home_id=msg["homeId"],
+            min_scheme_version=msg.get("minSchemeVersion", 0),
```

Comment:
Let's add a comment about when the scheme versions were included in this message.

Reply:
good idea